lib/tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
     "compilerOptions": {
         "strict": true,
-        "target": "es5",
-        "lib": ["dom", "es6", "dom.iterable", "scripthost"], // es6 stuff is polyfilled by stimulus
+        "target": "es6",
```

Comment:
I don't see an issue targeting `es6`, since our primary consumer (Stack Overflow's Core codebase) targets `es2019` in its tsconfig